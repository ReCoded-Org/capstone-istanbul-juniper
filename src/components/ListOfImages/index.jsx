import React from "react";
import { Row, Col } from "antd";

const ListOfImages = () => {
  // The cols in the return state are going to contain a list of clickable images
  // each image when clicked will show the puzzle of that specific image
  return (
    <Row>
      <Col>something</Col>
      <Col>something</Col>
      <Col>something</Col>
      <Col>something</Col>
    </Row>
  );
};

export default ListOfImages;
