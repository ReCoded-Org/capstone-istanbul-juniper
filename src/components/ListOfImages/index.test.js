import React from "react";
import ListOfImages from "./index.jsx";
import renderer from "react-test-renderer";

describe("ListOfImages Component", () => {
  it("matches the tree", () => {
    const tree = renderer.create(<ListOfImages />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
