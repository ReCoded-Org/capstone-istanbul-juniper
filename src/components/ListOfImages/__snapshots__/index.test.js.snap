// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListOfImages Component matches the tree 1`] = `
<div
  className="ant-row"
  style={Object {}}
>
  <div
    className="ant-col"
    style={Object {}}
  >
    something
  </div>
  <div
    className="ant-col"
    style={Object {}}
  >
    something
  </div>
  <div
    className="ant-col"
    style={Object {}}
  >
    something
  </div>
  <div
    className="ant-col"
    style={Object {}}
  >
    something
  </div>
</div>
`;
